saleor/graphql/csv/filters.py

```diff
@@ -5,19 +5,29 @@
 from ..utils.filters import filter_by_query_param
 
 
-def filter_created_by(qs, _, value):
+def filter_user(qs, _, value):
     user_fields = [
-        "created_by__pk",
-        "created_by__first_name",
-        "created_by__last_name",
-        "created_by__email",
+        "user__pk",
```

Comment:
Filtering by PK was here before but I'm wondering if this is useful - PKs are internal and are not known to the user so I'm not sure we should filter by them.